### PR TITLE
Feat/harrat/hello world

### PR DESCRIPTION
**Pull request by zohra051**
Hello world in my first page.

### CODE REVIEW

File: .bashrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export PATH=“[PATH_TO_FLUTTER_SDK_DIRECTORY]/flutter/bin:$PATH”
```
Comment (PiotrFLEURY) [inline .bashrc]: This file should not be in the project. please remove it.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (PiotrFLEURY) [inline .vscode/launch.json]: All files located in .vscode/ folder should be in the .gitignore and not be commit. Please remove this file from version control

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (PiotrFLEURY) [inline .vscode/tasks.json]: All files located in .vscode/ folder should be in the .gitignore and not be commit. Please remove this file from version control

File: bashrc [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export PATH=“[PATH_TO_FLUTTER_SDK_DIRECTORY]/flutter/bin:$PATH”
```
Comment (PiotrFLEURY) [inline bashrc]: This file should not be in the project. Please remove it.

File: lib/main.dart [hunk 5]
```diff
@@ -1,7 +1,15 @@
-import 'package:OutOfBounds/nve/nve-main-page.dart';
-import 'package:OutOfBounds/zht/zht-main-page.dart';
+//import 'package:OutOfBounds/nve/nve-main-page.dart';
```
Comment (PiotrFLEURY) [inline lib/main.dart]: Avoid comments commit. Remove unused imports instead of comment them.

File: lib/main.dart [hunk 6]
```diff
@@ -1,7 +1,15 @@
...
+//import 'package:OutOfBounds/zht/zht-main-page.dart';
 import 'package:flutter/material.dart';
+import './pages/home_page.dart';
 
+
+void main(){
```
Comment (PiotrFLEURY) [inline lib/main.dart]: Do NOT replace the main file content to stay compliant whith all the team contents. Please restore original file.

File: lib/pages/home_page.dart [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
```
Comment (PiotrFLEURY) [inline lib/pages/home_page.dart]: All your modifications should be located inside the lib/zht/ folder. Please move this file

File: lib/zht/zht-main-page.dart [hunk 8]
```diff
@@ -4,6 +4,9 @@ class ZhtMainPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     // TODO: implement build
```
Comment (PiotrFLEURY) [inline lib/zht/zht-main-page.dart]: Please remove TODO comment.

File: lib/zht/zht-main-page.dart [hunk 9]
```diff
@@ -4,6 +4,9 @@ class ZhtMainPage extends StatelessWidget {
...
+    return new Material(
```
Comment (PiotrFLEURY) [inline lib/zht/zht-main-page.dart]: This file should contain the Hello world message.

**Review by PiotrFLEURY — CHANGES_REQUESTED** [review]

File: .bashrc (hunk 1 above)
Comment (zohra051) [reply .bashrc]: Done.

**Review by PiotrFLEURY — APPROVED** [review]
LGTM!

**Pull request merged**